### PR TITLE
Update UI: Add Interview List UI

### PR DESCRIPTION
**Pull request by migfoo02**
Files added:
- InterviewCard
- InterviewListPanel

Changes:
- Updated JobDetailPanel to include interview list

### CODE REVIEW

File: src/main/java/seedu/application/ui/InterviewListPanel.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    public InterviewListPanel(ObservableList<Interview> interviewList) {
+        super(FXML);
+        interviewListView.setItems(interviewList);
+        interviewListView.setCellFactory(listView -> new InterviewListViewCell());
+    }
+
+    /**
+     * Custom {@code ListCell} that displays the graphics of a {@code Interview} using a {@code InterviewCard}.
+     */
+    class InterviewListViewCell extends ListCell<Interview> {
```
Comment (evanyan13) [inline src/main/java/seedu/application/ui/InterviewListPanel.java:34]: yeap the method to initialise InterviewListView looks good

**Review by evanyan13 — COMMENTED** [review]
Looks good for now. Maybe can push as an actual PR then we can merge to continue to work on the rest

**Review by czhiruo — APPROVED** [review]
LGTM

File: src/main/resources/view/JobDetailsPanel.fxml [hunk 2]
```diff
@@ -43,17 +44,17 @@
...
+            <Insets top="5" right="5" bottom="5" left="15" />
         </padding>
-        <Label fx:id="interviewTitle" styleClass="cell_big_label" text="\$interviews"/>
-        <Region minHeight="5" prefHeight="5" maxHeight="5"/>
-        <AnchorPane minHeight="200" prefHeight="200" prefWidth="195.0" styleClass="interviewListPanel"
-                    AnchorPane.bottomAnchor="0.0" AnchorPane.leftAnchor="0.0"
-                    AnchorPane.rightAnchor="0.0" AnchorPane.topAnchor="0.0" VBox.vgrow="ALWAYS">
-        </AnchorPane>
-    </VBox>
+        <StackPane fx:id="interviewListPanelPlaceHolder" minHeight="100" prefHeight="500"
```
Comment (evanyan13) [inline src/main/resources/view/JobDetailsPanel.fxml:54]: Good use of stackpane for a placeholder

**Review by evanyan13 — APPROVED** [review]
LGTM

**Pull request merged**